std/range/package.d

```diff
@@ -9676,20 +9705,33 @@ enum SearchPolicy
        of $(D trot), $(D gallop), $(D trotBackwards), and $(D
        gallopBackwards) strategies.
     */
-        binarySearch,
+    binarySearch,
 
     /**
        Similar to $(D trot) but starts backwards. Use it when
        confident that the value is around the end of the range.
     */
-        trotBackwards,
+    trotBackwards,
 
     /**
        Similar to $(D gallop) but starts backwards. Use it when
        confident that the value is around the end of the range.
     */
-        gallopBackwards
-        }
+    gallopBackwards
+}
+
+///
+@safe unittest
+{
+    import std.algorithm.comparison : equal;
+
+    auto a = assumeSorted([0, 1, 2, 3, 4, 5, 6, 7, 8, 9]);
+    auto p1 = a.upperBound!(SearchPolicy.binarySearch)(3);
+    assert(equal(p1, [4, 5, 6, 7, 8, 9]));
```

Comment:
Nit: FWIW people were complaining on the state of D survey about inconsistency in the docs. I think the majority uses the UFCS ( `p1.equal(X)`), so maybe a good idea to stick to that for public examples?